front/app/components/IdeaCard/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { memo, useEffect } from 'react';
+import React, { useEffect } from 'react';
 
```

Comment:
Just to confirm - the only change in this file was removing the "memo" right? :)

Reply:
Ah, and adding `className="e2e-idea-card-title"` to the `Title`. This, together with updating the class name in the test, is what fixed the test failure. Removing the memo kind of messed up the diff. 

I removed the memo because I thought it was causing the hot reload issue- in the end that was not the issue, but I think the memo is not necessary so I removed it anyway.

Reply:
Ok cool! Yeah sounds good :) Thanks!